### DISCUSSION TITLE
Add official support for Debian 12 (bookworm)

### DISCUSSION THREAD

**Discussion by cfranklin11** [google/or-tools#4023] [Feature requests]
**What language and solver does this apply to?**
All

**Describe the problem you are trying to solve.**
I'm trying to update from Debian 11 to Debian 12, but OR-Tools doesn't have an official binary installation for Debian 12 yet.

**Describe the solution you'd like**
Given that Debian 12 was released in June 2023 and Debian 11 will stop receiving security support in July 2024, it would be good to have binary files  to install that officially support Debian 12.

**Describe alternatives you've considered**
I was able to install the C++ binary for Debian 11 on a Debian 12 Docker image without error, and the application test suite passed, so I don't believe it broke my use of the GLOP solver. However, I haven't deployed this setup to production, and I don't know how using this binary in Debian 12 might affect other use cases.

**Comment by Mizux** [comment]
On my way to create it for our next release.

**Reply by kyliank** [reply]
Any updates on this!? :)

**Reply by Mizux** [reply]
completely forget it ^^;
Will create an issue from it and adapt our release process

**Comment by lperron** [comment]
Moved to bug.
